samples/excel/85-preview-apis/culture-info-date-time.yaml

```diff
@@ -0,0 +1,138 @@
+order: 5
+id: excel-culture-info-date-time
+name: 'Culture info: date and time'
+description: This sample shows how to apply the cultural settings APIs to retrieve read-only operating system date and time settings.
```

Comment:
It is still the operating system date/time when accessed through Excel for the Web? I think that's from the OneDrive/SharePoint site hosting the spreadsheet, but I could be wrong.

Reply:
Good question! How should I look into this? The feature spec says "This is based on current system settings" in the description for each of the new dateTimeFormat APIs. Would it be better if I simply dropped 'operating' from the sample description + other references in this sample?

Reply:
Yeah, that seems like a reasonable solution.